purchase_order_product_summary/report/purchase_order_summary_template.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="report_purchaseorder_document_product_summary" inherit_id="purchase.report_purchaseorder_document">
+        <xpath expr="//div[@id='total']" position="after">
+            <div class="row justify-content-start">
+                <div class="col-4">
+                    <h4>Summary Totals:</h4>
+                </div>
+            </div>
+            <table class="table table-sm o_main_table">
+                <thead>
+                    <tr>
+                        <th name="th_id"><strong>Code</strong></th>
+                        <th name="th_name"><strong>Product Name</strong></th>
```

Comment:
@lmiguens-solvos what do you prefer? Separed columns, or both code and product in the same column?

Reply:
I really don't care. 
I don't really know what will be better. Seeing that the product codes are all the same length ... well, it doesn't matter.

Reply:
got it :)